internal/app/wwctl/container/list/main.go

```diff
@@ -28,6 +28,10 @@ func CobraRunE(vars *variables) func(cmd *cobra.Command, args []string) (err err
 				for i := 0; i < len(containerInfo); i++ {
 					createTime := time.Unix(int64(containerInfo[i].CreateDate), 0)
 					modTime := time.Unix(int64(containerInfo[i].ModDate), 0)
+					if vars.utc {
+						createTime = createTime.UTC()
+						modTime = modTime.UTC()
+					}
```

Comment:
I realized we don't actually need to add this; we can just set `TZ=UTC` when running the test suite.

I'll try to add a commit that does that for your comment.